packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -77,13 +87,61 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
     } = useComboboxPrimitive({
       initialSelectedItem,
       items,
-      itemToString: itemToString || defaultItemToString,
       onHighlightedIndexChange,
       onInputValueChange,
       onIsOpenChange,
       onSelectedItemChange,
+      ...(itemToString && {itemToString}),
+      ...(initialIsOpen && {initialIsOpen}),
+      ...(inputValue && {inputValue}),
+      ...(selectedItem && {selectedItem}),
```

Comment:
What happens if selectedItem and initialSelectedItem are different on mount? 🤔

Reply:
I guess the same question could be asked of Downshift directly, it's just a pass through replicating their API. 🤔